frontend/src/Commons/Styles/Header.style.ts

```diff
@@ -48,12 +55,16 @@ const Title = styled.h1`
   margin: 50px 0 20px 0;
   line-height: 54px;
 
+  @media (min-width: ${screenSizes.tablet}) and (max-width: ${screenSizes.desktop}) {
+    font-size: 40px;
+    height: 100%;
+    width: 100%;
+  }
   @media (min-width: ${screenSizes.desktop}) {
     order: 2; /* Moves text after logo */
-    font-size: 48px;
+    font-size: 5vh;
     margin: 0;
-    position: absolute;
-    left: 40vw;
+    margin-left: -19vw;
```

Comment:
Centering is still being hardcoded here. I recommend using a flexbox to make life easier. It looks slightly off-centered:

![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/80939533/9a2c6949-950a-4631-96f0-1ca5d096281a)

Reply:
Centering is tough. 😢

Reply:
I will leave it as is for now. Let's not focus on getting the centerness perfect.